class.application-passwords.php

```diff
@@ -228,6 +244,56 @@ public static function create_new_application_password( $user_id, $name ) {
 	}
 
 	/**
+	 * Create application password via ajax
+	 *
+	 * @since 0.1-dev
+	 *
+	 * @access public
+	 * @static
+	 */
+	public static function ajax_create_application_password() {
+		$user_id = get_current_user_id();
```

Comment:
Can an admin user create/delete app passwords for other users? If so, then we can't reliably use `get_current_user_id()`.